### DISCUSSION TITLE
Data refreshing is not working

### DISCUSSION THREAD

**Discussion by rafidahsanSysable** [nuxt/nuxt#16487] [Questions]
I have a similar case like this
https://v3.nuxtjs.org/getting-started/data-fetching#refreshing-data

but my value on fetch api doesn't change 

```
let pageNumber = ref(1)

const { data:result, refresh } = await useFetch(api()+'/vendor/patients?page='+pageNumber.value, {
  headers: {
    Authorization: `Bearer ${authStore.getToken()}`
  },
})
  
patients = proxyToArrayConvert(result.value.data)
patientStore.setPatients(patients)

const page = (page) => {
  pageNumber.value = page
  refresh()
}
```

Page is being called but the value is not changed in fetch api.. Thanks in advance

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏
